src/IotHub/IotHub/help/New-AzIotHubKey.md

```diff
@@ -0,0 +1,161 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.IotHub.dll-Help.xml
+Module Name: Az.IotHub
+online version: https://docs.microsoft.com/en-us/powershell/module/az.iothub/new-aziothubkey
+schema: 2.0.0
+---
+
+# New-AzIotHubKey
+
+## SYNOPSIS
+Generate an Azure IoT Hub key.
+
+## SYNTAX
+
+```
+New-AzIotHubKey [-ResourceGroupName] <String> [-Name] <String> [-KeyName] <String> [-RenewKey] <String>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Generate an Azure IoT Hub key.
+
+## EXAMPLES
+
+### Example 1 Regenerate primary key
+```
+PS C:\> New-AzIotHubKey -ResourceGroupName "myresourcegroup" -Name "myiothub" -KeyName "testKey" -RenewKey "primary"
```

Comment:
@anusapan for each example, please provide the corresponding output that the user would see if they were to run the command

Reply:
Added output